metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/dao/WorkflowUtils.java

```diff
@@ -102,41 +111,58 @@ public WorkflowUtils(WorkflowExecutionDao workflowExecutionDao) {
       throw new BadContentException("Workflow should not be empty.");
     }
 
-    // Compile the list of enabled plugins.
-    final List<AbstractExecutablePluginMetadata> plugins = workflow.getMetisPluginsMetadata()
+    // Compile the list of enabled enabledPlugins.
```

Comment:
That's the danger of intellij renaming with comment strings. Did this happen in other places?

Reply:
Indeed this happens some times..